homeassistant/components/bluetooth/strings.json

```diff
@@ -2,6 +2,9 @@
   "config": {
     "flow_title": "{name}",
     "step": {
+      "enable_bluetooth": {
+        "description": "This integration allows Home Assistant to discover Bluetooth devices and share the Bluetooth adapter with `bleak` based integrations. If you find your legacy integrations need exclusive use of the Bluetooth adapter, deleting this integration will release control of the adapter."
```

Comment:
I think that this description is better fit for the docs. Especially Bleak is too technical

Reply:
Moving to the docs